### PR TITLE
Fixes #2372 prompt footer in summary

### LINKED ISSUE #2372: Prompt footer added to cards in retro summary
### Acceptance Criteria
- [ ] The reflection cards in the retro meeting summary have the prompt footer. A quick style pass in code is sufficient.

### Estimated Effort
- 10.5 points ([see CONTRIBUTING.md](https://github.com/ParabolInc/action/blob/master/CONTRIBUTING.md#points-and-sizes))

### PR DESCRIPTION
**Pull request by ackernaut**
#### Test
- [ ] Run a retro locally with at least 1 voted topic. See the prompt question in the footer of the reflections in the summary view (browser, email).

### CODE REVIEW

**Comment by ackernaut** [comment]
@mattkrick I’m stumped, trying to figure out why I can’t end a meeting. I was able to see the changes on an existing meeting summary only. My guess is I haven’t updated the meeting data that gets used to end the meeting and send the summary. Here is my error (and I’ve searched through a lot of files without success):
```
{ TypeError: Cannot destructure property `question` of 'undefined' or 'null'.
    at content (/Users/terryacker/code/parabolinc/action/src/universal/modules/email/components/RetroDiscussionTopics/RetroDiscussionTopic.js:111:36)
    at Array.map (<anonymous>)
    at map (/Users/terryacker/code/parabolinc/action/src/universal/modules/email/components/RetroDiscussionTopics/RetroDiscussionTopic.js:111:26)
    at Array.map (<anonymous>)
    at map (/Users/terryacker/code/parabolinc/action/src/universal/modules/email/components/RetroDiscussionTopics/RetroDiscussionTopic.js:108:23)
    at processChild (/Users/terryacker/code/parabolinc/action/node_modules/react-dom/cjs/react-dom-server.browser.development.js:2408:14)
    at resolve (/Users/terryacker/code/parabolinc/action/node_modules/react-dom/cjs/react-dom-server.browser.development.js:2334:5)
    at ReactDOMServerRenderer.render (/Users/terryacker/code/parabolinc/action/node_modules/react-dom/cjs/react-dom-server.browser.development.js:2656:22)
    at ReactDOMServerRenderer.read (/Users/terryacker/code/parabolinc/action/node_modules/react-dom/cjs/react-dom-server.browser.development.js:2630:23)
    at Object.renderToStaticMarkup (/Users/terryacker/code/parabolinc/action/node_modules/react-dom/cjs/react-dom-server.browser.development.js:3032:25)
    at _renderTemplate (/Users/terryacker/code/parabolinc/action/node_modules/oy-vey/lib/utils/Renderer.js:38:38)
    at Object.renderTemplate (/Users/terryacker/code/parabolinc/action/node_modules/oy-vey/lib/utils/Renderer.js:53:32)
    at renderTemplate (/Users/terryacker/code/parabolinc/action/src/universal/modules/email/components/SummaryEmail/newMeetingSummaryEmailCreator.js:27:14)
    at emailFactory (/Users/terryacker/code/parabolinc/action/src/server/email/sendEmail.js:33:52)
    at makeMailgunApiData (/Users/terryacker/code/parabolinc/action/src/server/email/sendEmail.js:103:23)
    at sendNewMeetingSummary (/Users/terryacker/code/parabolinc/action/src/server/graphql/mutations/helpers/endMeeting/sendNewMeetingSummary.js:63:30)
  message:
   'Cannot destructure property `question` of \'undefined\' or \'null\'.',
  locations: [ { line: 4, column: 3 } ],
  path: [ 'endNewMeeting' ] }
```

**Comment by mattkrick** [comment]
i think you'll need to also add that to the rethinkdb query (sendNewMeetingSummary).
i haven't added SSR to relay stuff yet, so data for email components have to be updated on the server

File: src/server/graphql/mutations/createReflection.js [hunk 1]
```diff
@@ -74,6 +74,7 @@ export default {
       content: normalizedContent,
       isActive: true,
       meetingId,
+      phaseItem,
```
Comment (ackernaut) [inline src/server/graphql/mutations/createReflection.js]: @mattkrick adding the `phaseItem` object here seems to do the trick, I think this needed to be in the payload that is used to end the meeting and send the summary

**Comment by ackernaut** [comment]
@mattkrick ready for your review!

**Pull request merged**